frontend/src/modules/app/layouts/ScreensaverRoute.tsx

```diff
@@ -90,6 +103,9 @@ const ContentComponent = React.memo(({ children }: PropsWithChildren<unknown>) =
     setShowBorder(notifyAlarm);
   }, [notifyAlarm]);
```

Comment:
What does this `useEffect` do, or what is it supposed to do?

Reply:
It listens for Active alarm event and adds a red border around the page. Adding following lines for it
```
  /**
   * Triggers when Alarm event is active (Referenced in `OverlayScreen` )
   * RED_BORDER reduxs store is dispatched when alarm is active
   * It adds a red border around the page
   */
```